src/guide/bash.md

```diff
@@ -562,11 +562,63 @@ available on the network:
 ]
 ```
 
+::: info
+
 Iroha 2 currently doesn't validate the account names, so you could (in
 theory) add invalid characters to the name, e.g. spaces. **We recommend
 sticking to English alphanumeric characters and underscores**.
 
-## 6. Visualizing outputs
+:::
+
+## 6. Transferring assets
+
+After minting the assets, you can transfer them to another account. To be
+able to do this, you have to grant [permission](advanced/permissions.md) to
+transfer assets. Let's look at how this process works.
+
+First, create a file named `permission_token.json` with the permission you
+want to grant. In our case, we want Mad Hatter to be able to transfer `tea`
+asset.
+
+```json
+{
+  "name": "can_transfer_user_assets",
```

Comment:
```suggestion
    "definition_id": {
    "name": "can_transfer_user_assets"
  },
```

Reply:
Without that `PermissionToken` deserialization will fail

Reply:
Probably someday we should refactor things like that. So that you will be able to write

```json
{
  "definition_id" : "can_transfer_user_assets",
  ...
}
```

Reply:
Created [iroha/#2756](https://github.com/hyperledger/iroha/issues/2756)